releases.md

```diff
@@ -11,12 +11,12 @@ The following table lists in-support .NET releases.
 
 |  Version  | Release Date | Support | Latest Patch Version | End of Support |
 | :-- | :-- | :-- | :-- | :-- |
-| [.NET 6](release-notes/6.0/README.md) | [November, 2021](https://devblogs.microsoft.com/dotnet/announcing-net-6-preview-3/) | [Preview][policies] | [6.0 Preview 3][6.0 Preview 3] | November, 2024 |
+| [.NET 6](release-notes/6.0/README.md) | [November, 2021](https://devblogs.microsoft.com/dotnet/announcing-net-6-preview-3/) | [Preview][policies] | [6.0 Preview 5][6.0 Preview 5] | November, 2024 |
```

Comment:
It's a bit weird to have an end of support date for .NET 6 since in theory it's not supported yet

```suggestion
| [.NET 6](release-notes/6.0/README.md) | [November, 2021](https://devblogs.microsoft.com/dotnet/announcing-net-6-preview-5/) | [Preview][policies] | [6.0 Preview 5][6.0 Preview 5] | November, 2024 |
```

Reply:
Yes, it is. I think we followed a similar pattern with .NET 5 though I don't recall when we first published the expected eol date.